Answers.md

```diff
@@ -0,0 +1,23 @@
+- find all customers that live in London. Returns 6 records.
```

Comment:
good job on these commands. 

I will say... might not use these too often but they are super useful and good to know. I highly recommend looking for other classes or opportunities to keep up on your SQL skills. It's very marketable.